### ISSUE TITLE
Huge battery drain by Android System after latest update

### ISSUE THREAD

**Issue by ArtikusHG** [GrapheneOS/os-issue-tracker#2043]
I am using a Pixel 4a with GrapheneOS and after the most recent update battery life has been terrible. Before my phone could last almost 2 days. Now it's losing battery at a rate of about 1% every 2 minutes. I have restricted all background activity on apps that I used to have in the background, removed some apps I recently installed, rebooted, etc - nothing helps. 
![Screenshot_20230319-000410](https://user-images.githubusercontent.com/24320212/226142753-ceab9810-562f-4313-88e5-075ca3ef8dd5.png)

**Comment by thestinger** [comment]
You're using at least one app that's draining a substantial amount of power (Fork Client).

**Comment by thestinger** [comment]
Restricting background usage won't stop apps from running a foreground service.

You'll need to resolve the app / configuration issues causing power drain. It's highly unlikely that it's caused by any OS bug and the small amount of information that you're providing has clear evidence of it being an issue with a specific app. The OS battery usage summary is usually not very good at identifying the problem apps, but in this case it did identify an app that's draining a lot of power.

**Comment by ArtikusHG** [comment]
Forkgram did drain a lot of power before, true, but Android System was much, much lower, as well as phone idle, and that was *before* I restricted the battery for Forkgram. I still believe it is the latest update - it started after I updated my phone, and is showing that not an app, but Android System is using battery. Just as I type this in Mull, with Forkgram killed, battery went down by 2%. This is not normal.
